ci-operator/jobs/coreos/rpm-ostree/coreos-rpm-ostree-master-presubmits.yaml

```diff
@@ -4,24 +4,24 @@ presubmits:
     always_run: true
     branches:
     - master
-    cluster: build01
-    context: ci/prow/build
+    cluster: build02
```

Comment:
Though this test (and clang-analyzer) doesn't actually need virt, right? Isn't it better to leave that cluster for workloads which need virt?

Reply:
We can't currently distinguish unfortunately.

Reply:
You mean, for a given repo, all the jobs have to run on the same cluster?

Reply:
Yeah currently AIUI.